packages/data/src/redux-store/metadata/selectors.js

```diff
@@ -20,9 +25,9 @@ import { selectorArgsToStateKey } from './utils';
  * @param {string}     selectorName Selector name.
  * @param {unknown[]?} args         Arguments passed to selector.
  *
- * @return {boolean | undefined} isResolving value.
+ * @return {StateValue | undefined} isResolving value.
  */
-export function getIsResolving( state, selectorName, args ) {
+export function getResolutionState( state, selectorName, args ) {
```

Comment:
Nit: we can also have a user-facing API called `getResolutionStatus` to indicate which `status`(`StatusKey`) the resolution is in.

Reply:
I'd rather not introduce too many public-facing selectors in this PR. Let's take it into a follow-up one